src/geometry/euclidean/basic.lean

```diff
@@ -499,7 +499,7 @@ by rw [dist_left_midpoint p1 p2, dist_right_midpoint p1 p2]
 /-- If M is the midpoint of the segment AB, then ∠AMB = π. -/
 lemma angle_midpoint_eq_pi (p1 p2 : P) (hp1p2 : p1 ≠ p2) : ∠ p1 (midpoint ℝ p1 p2) p2 = π :=
 have p2 -ᵥ midpoint ℝ p1 p2 = -(p1 -ᵥ midpoint ℝ p1 p2), by { rw neg_vsub_eq_vsub_rev, simp },
-by simp [angle, this, hp1p2]
+by simp [angle, this, hp1p2, -zero_lt_one]
```

Comment:
What goes wrong without this?

Reply:
it goes down a different path involving `angle_smul_left_of_pos`, but it just happens to be the wrong one for this

Reply:
also, I didn't realise how heavy duty the machinery is to do basic euclidean geometry, my god

Reply:
It's exactly that, simp goes another path if it can exploit that `0 < 1`, which does not end at the same place (and does not complete the proof).